app/test_main.py

```diff
@@ -1 +1,56 @@
-# write your code here
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+from typing import Callable
+from unittest.mock import MagicMock
+from app.main import can_access_google_page
+
+
+@pytest.fixture
+def url() -> str:
```

Comment:
you don't really need this fixture - you use it only in one place and since your functions are mocked it doesn't really matter what url is